akka-persistence-typed/src/main/scala/akka/persistence/typed/javadsl/PersistentBehavior.scala

```diff
@@ -18,7 +18,7 @@ import scala.util.{ Failure, Success }
 
 /** Java API */
 @ApiMayChange
-abstract class PersistentBehavior[Command, Event, State >: Null] private (val persistenceId: PersistenceId, supervisorStrategy: Option[BackoffSupervisorStrategy]) extends DeferredBehavior[Command] {
+abstract class PersistentBehavior[Command, Event, State >: Null] private[akka] (val persistenceId: PersistenceId, supervisorStrategy: Option[BackoffSupervisorStrategy]) extends DeferredBehavior[Command] {
```

Comment:
Does this constructor really become private from Java?

Reply:
probably not, but it doesn't matter because that Option should be changed to Optional (think I did that in other PR)